# HG changeset patch
# User Rik <rik@octave.org>
# Date 1479428005 28800
#      Thu Nov 17 16:13:25 2016 -0800
# Node ID 10a7eae901dcb64f5e2e0f54b66a54560a3f802b
# Parent  f403ad463aa4cb0d19cba1115ea7a02072002a30
isocolors.m: Fix leftover cruft in docstring from cset f403ad463aa4.

* isocolors.m: Remove @example opening block now that example is a demo.

diff --git a/scripts/plot/draw/isocolors.m b/scripts/plot/draw/isocolors.m
--- a/scripts/plot/draw/isocolors.m
+++ b/scripts/plot/draw/isocolors.m
@@ -43,20 +43,16 @@
 ## @var{y}, @var{z} are not given then @code{meshgrid} computed values are
 ## used.
 ##
 ## Optionally, a patch handle @var{p} can be given as the last input argument
 ## to all function call variations and the vertex data will be extracted
 ## from the isosurface patch object.  Finally, if no output argument is given
 ## then the colors of the patch given by the patch handle @var{p} are changed.
 ##
-## For example:
-##
-## @example
-##
 ## @seealso{isosurface, isonormals}
 ## @end deftypefn
 
 ## Author: Martin Helm <martin@mhelm.de>
 
 function cdat = isocolors (varargin)
 
   calc_rgb = false;
